### DISCUSSION TITLE
Welcome to Hysteresis Discussions!

### DISCUSSION THREAD

**Discussion by cslotboom** [cslotboom/Hysteresis#5] [General]
## Welcome!
  This is a place to discuss general usage of the hysteresis Package. In this hub feel free to:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded.

If you have questions, please submit source code with those questions!

**Comment by aberu025** [comment]
Hey thanks for the great package! Will be doing about 450 tests on CLT connections in January and couldn't find something to expedite the analysis for finding backbone and EEEP until I found your package. I'm sure it will be super useful. Cheers

**Comment by cslotboom** [comment]
Hey! Glad to hear the package has been useful for you so far! As always feel free to reach out here if you have any questions, comments, or requests. 

One comment, the ASTM E2126 EEEP fitting function I wrote is based on a document that appears to no longer be public (it was here: http://www.materialstandard.com/wp-content/uploads/2019/10/E2126-11.pdf).
My understanding is this was done to the 2019 standard, but it's probably worth a double check to make sure you're getting the same results. I'm happy to work with you there if there are any tweaks needed.

450 is a lot of tests - sounds like an exciting research program!

**Reply by aberu025** [reply]
Working great for now! I'm going through the package slowly with my data and works well for now! Although I have to fix units lol 

![image](https://github.com/cslotboom/hysteresis/assets/152050842/339d83f6-8a69-4d34-9cbf-85206c2224ba)

**Comment by hirottawa** [comment]
Hello,
Thank you for this useful tool. Could you please explain how to use it as I don not have knowledge (I am new in this website).
Best

**Comment by hirottawa** [comment]
Hello again,
I think now I can run the code with hours of research on the subject (around 6 hours starting from time I posted my first message...). 
But I have an issue, I tried to fix it but without success. This is the message I got 
Exception has occurred: FileNotFoundError
xyTestData.csv not found.

Could you please help on this ?
Thank you
